modules/core/src/main/java/org/apache/ignite/internal/processors/query/QueryUtils.java

```diff
@@ -578,6 +581,9 @@ public static QueryTypeCandidate typeForQueryEntity(GridKernalContext ctx, Strin
                     ((GridCacheDefaultAffinityKeyMapper)keyMapper).affinityKeyPropertyName(desc.keyClass());
 
                 if (affField != null) {
+                    if (desc.aliases().containsKey(affField))
+                        affField = desc.aliases().get(affField);
```

Comment:
```suggestion
                    affField = desc.aliases().getOrDefault(affField, affField);
```

Reply:
@AMashenkov , hi! Thanks for review. Applied both suggestions with an even additional improvement.